core/geometry/src/curve/Arc3d.ts

```diff
@@ -226,6 +226,35 @@ export class Arc3d extends CurvePrimitive implements BeJSONFunctions {
       result,
     );
   }
+  /**
+   * Create an arc from three points on the ellipse: two points on an axis and one in between.
+   * @param point0 start of arc, on an axis
+   * @param point1 point on arc somewhere between `point0` and `point2`
+   * @param point2 point on arc directly opposite `point0`
+   * @param sweep angular sweep, measured from `point0` in the direction of `point1`.
+   *  For a semicircle from `point0` to `point2` passing through `point1`, pass `AngleSweep.createStartEndDegrees(0,180)`.
+   *  Default value is full sweep to create the entire ellipse.
+   * @param result optional preallocated result
+   * @returns elliptical arc, or undefined if construction impossible.
+   */
+  public static createStartMiddleEnd(point0: XYAndZ, point1: XYAndZ, point2: XYAndZ, sweep?: AngleSweep, result?: Arc3d): Arc3d | undefined {
+    const center = Point3d.createAdd2Scaled(point0, 0.5, point2, 0.5);
```

Comment:
Interesting, I would have used interpolate here...is createAdd2Scaled preferable/more efficient?